packages/combinator/src/components/combinator.vue

```diff
@@ -29,13 +10,13 @@
     >
       May have unexpected behaviour.
     </dt-notice>
-    <div class="d-d-flex d-fl-grow1 d-hmx464">
+    <div class="d-d-flex d-hmx464">
       <div
         class="dtc-root d-d-grid d-of-hidden d-ba d-bar4 d-w100p"
         :class="{
           [rootClass]: true,
-          [`dtc-root--sidebar-${settings.root.sidebar}`]: true,
-          'dtc-root--blueprint': blueprint,
+          [`dtc-root&#45;&#45;sidebar-${settings.root.sidebar}`]: true,
+          'dtc-root&#45;&#45;blueprint': blueprint,
```

Comment:
Why is this necessary?

Reply:
Didn't put too much attention on those, removed and updated those classes to be more readable.

Reply:
I just mean the `&#45;&#45;`

Reply:
Yeah, those are `-` maybe some linter converted them